src/dropdown/dropdown.ts

```diff
@@ -1,12 +1,12 @@
 import {Directive, Input, HostListener} from 'angular2/core';
 
-@Directive({selector: '[ngb-dropdown]', exportAs: 'ngbDropdown', host: {'[class.open]': 'open'}})
+@Directive({selector: '[ngbDropdown]', exportAs: 'ngbDropdown', host: {'[class.open]': 'open'}})
```

Comment:
That was a breaking change as far as I understand - it makes me sad, as it is more natural to understand than this camelCase :( .